README.md

```diff
@@ -1,81 +1,104 @@
+[![Apache-2.0][licence-badge]][licence-file]
 
-The Iterative Provider makes it easy to:
+TPI is a [Terraform](https://terraform.io) plugin built with machine learning in mind. Full lifecycle management of computing resources (including GPUs and respawning spot instances) from several cloud vendors (AWS, Azure, GCP, K8s)... without needing to be a cloud expert.
 
-- Rapidly move local machine learning experiments to a cloud infrastructure
-- Take advantage of training models on spot instances without losing any progress
-- Unify configuration of various cloud compute providers
-- Automatically destroy unused cloud resources (compute instances are terminated on job completion/failure, and storage is removed when results are downloaded)
+- **Provision Resources**: create cloud compute (CPU, GPU, RAM) & storage resources without reading pages of documentation
+- **Sync & Execute**: easily sync & run local data & code in the cloud
+- **Low cost**: transparent auto-recovery from interrupted low-cost spot/preemptible instances
+- **No waste**: auto-cleanup unused resources (terminate compute instances upon job completion/failure & remove storage upon download of results)
+- **No lock-in**: switch between several cloud vendors with ease due to concise unified configuration
 
-The Iterative Provider can provision resources with the following cloud providers and orchestrators:
+Supported cloud vendors include:
 
-- Amazon Web Services
+- Amazon Web Services (AWS)
 - Microsoft Azure
-- Google Cloud Platform
-- Kubernetes
+- Google Cloud Platform (GCP)
+- Kubernetes (K8s)
 
-## Documentation
+## Usage
 
-See the [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) guide to learn how to use the Iterative Provider. More details on configuring and using the Iterative Provider are in the [documentation](https://registry.terraform.io/providers/iterative/iterative/latest/docs).
+### Requirements
 
-## Support
+- [Install Terraform 1.0+](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform), e.g.:
+  + Brew (Homebrew/Mac OS): `brew tap hashicorp/tap && brew install hashicorp/tap/terraform`
+  + Choco (Chocolatey/Windows): `choco install terraform`
+  + Conda (Anaconda): `conda install -c conda-forge terraform`
+  + Debian (Ubuntu/Linux):
+    ```
+    sudo apt-get update && sudo apt-get install -y gnupg software-properties-common curl
+    curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
+    sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+    sudo apt-get update && sudo apt-get install terraform
+    ```
+- Create an account with any supported cloud vendor and expose its [authentication credentials via environment variables](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/authentication)
```

Comment:
the link is broken

Re the text. It might be better to specify that it is about AWS/Azure/GCP. Initially I was thinking about cloud account in Terraform 😅 Also, it's not enough to have an account, user needs to set up credentials.

Any ideas for shortcuts here? Let's assume a user has credentials already, how to onboard them faster? How user can check if credentials set up properly?